hazelcast/src/main/java/com/hazelcast/map/impl/nearcache/invalidation/Invalidation.java

```diff
@@ -30,7 +30,7 @@
 /**
```

Comment:
We need to get confirmation from e.g. the client or near cache team to see if this class is used on the client.